modules/installation-configuration-parameters.adoc

```diff
@@ -1134,14 +1162,30 @@ The `licenses` parameter is a deprecated field and nested virtualization is enab
 |For control plane machines, the ID of the project in which the KMS key ring exists. This value defaults to the VM project ID if not set.
 |The GCP project ID.
 
-////
-`controlPlane.platform.gcp.osDisk.encryptionKey.kmsKeyServiceAccount`
+|`controlPlane.platform.gcp.osDisk.encryptionKey.kmsKeyServiceAccount`
+|The GCP service account used for the encryption request for control plane machines. If absent, the Compute Engine default service account is used. For more information on GCP service accounts, see Google's documentation on link:https://cloud.google.com/compute/docs/access/service-accounts#compute_engine_service_account[service accounts].
+|The GCP service account email, for example `<service_account_name>@<project_id>.iam.gserviceaccount.com`.
 
-The GCP Compute Engine System service account used for the encryption request for the given KMS key. The Compute Engine default service account is always used for control plane machines during installation, which follows this pattern: `service-<project_number>@compute-system.iam.gserviceaccount.com`. The default service account must have access to the KMS key specified for the control plane machines. The custom service account defined is available for use during post-installation operations. For more information on GCP service accounts, see Google's documentation on link:https://cloud.google.com/iam/docs/service-accounts#types[Types of service accounts].
+|`controlPlane.platform.gcp.osDisk.diskSizeGB`
+|The size of the disk in gigabytes (GB). This value applies to control plane machines.
+|Any size between 16 GB and 65536 GB.
```

Comment:
Is this a string? Integer? Other?

Reply:
Reworded as "Any integer between 16 and 65536."